### PR TITLE
Adding tatc

### PR DESCRIPTION
**Pull request by ptgrogan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ptgrogan** [comment]
I am willing to be listed as maintainer.

**Comment by ptgrogan** [comment]
@conda-forge-admin, please ping team

File: recipes/tatc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% set name = "tatc" %}
+{% set version = "3.1.0.beta1" %}
```
Comment (ocefpaf) [inline recipes/tatc/meta.yaml:2]: This is not a stable release and falls under: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Comment (ptgrogan) [reply recipes/tatc/meta.yaml:2]: Thanks for identifying this; I will work to make a full release in the coming days.

Comment (ptgrogan) [reply recipes/tatc/meta.yaml]: This recipe has been updated for a stable release.

**Comment by ptgrogan** [comment]
@conda-forge-admin, please ping team

File: recipes/tatc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  url: https://github.com/code-lab-org/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3261a434e027eaeae0bd5df21482f8bbeae634d8d6546c77e82621ac7bbf55ad
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/tatc/meta.yaml:18]:
```suggestion
    - python >=3.6
```

File: recipes/tatc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    - geopandas
+    - numba
+    - numpy
+    - pandas
+    - pip
+    - pydantic
+    - pyproj
+    - shapely
+    - skyfield
+    - sgp4
```
Comment (ocefpaf) [inline recipes/tatc/meta.yaml:28]:
```suggestion
    - pip
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]
This recipe can probably be noarch and, unless it imports itself at build time, you don't need the runtime deps on host.

**Review by ptgrogan — COMMENTED** [review]
all changes accepted; thanks for identifying the configuration changes

**Pull request merged**